src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(number1, number2) {
```

Comment:
It looks good, but I would suggest just using "a, b" as variables. By the name of methods, it's already clear, that you will pass integers as parameters.